c/cppzmq/cppzmq-4.8.1.eb

```diff
@@ -0,0 +1,30 @@
+# ll4strw Lorentz Institute
+easyblock = 'CMakeMake'
+
+name = 'cppzmq'
+version = '4.8.1'
+
+homepage = 'https://github.com/zeromq/cppzmq'
+description = "cppzmq is a C++ binding for libzmq."
+
+toolchain = SYSTEM
```

Comment:
Same as for `Catch2`, this is going to be painful in the long run, so although there's no technical reason for having a non-system toolchain, it will be way easier in practice because it requires `CMake` to install...